docs/helpers.rst

```diff
@@ -0,0 +1,53 @@
+Resource helpers
+================
+One of the core concepts of Seaworthy is the concept of a resource helper.
+These are wrappers around the resource models provided by the `Docker SDK for
+Python`_. There is a helper class for each of the basic Docker resource types:
+
+- :class:`~seaworthy.helper.ImageHelper` manages
+  :class:`~docker.models.images.Image` resources
```

Comment:
Would it make sense to drop the leading `~` from the docker-py references so that it's clearer that they're not part of Seaworthy?

Reply:
Let me try it and see how it looks.